CONTRIBUTING.md

```diff
@@ -105,11 +105,9 @@ Code all lives in the [dotnet/samples](https://github.com/dotnet/samples) reposi
 
 - *snippets* contains the code snippets used throughout the documentation. Inside the *snippets* folder, the folder names identify the language of the code snippet.
 
-We will expand these directories as the docs repository adds new content. For example, we will add Xamarin directories, like `xamarin-ios` and `xamarin-android` directories.
+Under these top-level folders, the samples structure mirrors the folder structure of the related articles. For example, **core/get-started** contains the samples that relate to the articles in the **docs/core/get-started** folder. Snippets for the C# Language keywords are in **snippets/csharp/reference/keywords**.
```

Comment:
@BillWagner those are not the best examples:
- There is no **core/get-started** folder. It's [core/getting-started](https://github.com/dotnet/samples/tree/master/core/getting-started)
- There is no **docs/core/get-started** folder. There is [get-started.md file](https://github.com/dotnet/docs/blob/master/docs/core/get-started.md). However, above mentioned samples look to serve [docs/core/testing](https://github.com/dotnet/docs/tree/master/docs/core/testing) folder
- Snippets for the C# keywords are all over the samples repo. For example, check the snippet location for the [ascending.md file](https://raw.githubusercontent.com/dotnet/docs/master/docs/csharp/language-reference/keywords/ascending.md). Though, I see and follow the tendency to place new C# keyword snippets to the [snippets/csharp/keywords](https://github.com/dotnet/samples/tree/master/snippets/csharp/keywords) folder (I guess, we forgot about the "language-reference" subfolder when created that project)

So, what would be a good example? Something recently created. For example, [machine-learning/tutorials](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials) in the **samples** repo support [docs/machine-learning/tutorials](https://github.com/dotnet/docs/tree/master/docs/machine-learning/tutorials) in the **docs** repo.

As for snippets, there is [snippets/csharp/how-to](https://github.com/dotnet/samples/tree/master/snippets/csharp/how-to) and [docs/csharp/how-to](https://github.com/dotnet/docs/tree/master/docs/csharp/how-to) pair.